python/pipeline/experiment.py

```diff
@@ -333,6 +333,73 @@ def plot_calibration_curve(self):
         return fig
 
 
+@schema
+class SurgeryTypes(dj.Lookup):
```

Comment:
Just as a convention in the lab, the class name reflects the type of entries in each row of the table. So in this case it should be `SurgeryType`, etc in the singular form.  The same goes for other tables.

Reply:
Got it! I've updated the class names.